src/pages/TeamOverview/TeamOverview.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import {useLocation, useParams} from 'react-router-dom';
+import {ListItem, UserData} from 'types/types';
+import {getUserColumns} from 'utils/utils';
+import {getTeamOverview, getUserData} from '../../api/api';
+import Card from '../../components/Card/Card';
+import {GlobalContainer} from '../../components/global.styled';
+import Header from '../../components/Header/Header';
+import List from '../../components/List/List';
+
+const getUsersCards = (users: UserData[]): ListItem[] => {
+    return users.map((user) => {
+        var columns = getUserColumns(user);
+        
+        return {
+            id: user.id,
+            url: `/user/${user.id}`,
+            columns,
+            navigationProps: user,
+        };
+    }) as ListItem[];
+};
+
+const getTeamLeadCard = (teamLead: UserData): JSX.Element => {
```

Comment:
changed the method name from `mapTLead` to `getTeamLeadCard` , so it can be better understood the method objective, also using variable `teamLead` instead of `tLead`, and specify method type